Gemfile

```diff
@@ -18,6 +18,7 @@ when /3/
 else
   gem 'mongoid', version
 end
+gem 'mongoid-compatibility'
```

Comment:
This needs to become a .gemspec dependency, otherwise it will be failing at runtime.

Reply:
I believe it's already in the .gemspec file. Is this not it? https://github.com/mongoid/mongoid-history/blob/master/mongoid-history.gemspec#L23

Reply:
Or are you saying to add a version dependency so that it works with version 0.5.1 or above?

Reply:
Then you don't need to add it here, don't you?